lib/ContentType/ACF/Group.php

```diff
@@ -69,13 +69,13 @@ private function get_location() : array {
 		$location = [];
 		$params   = $this->get_locations();
 
-		foreach ( $params as $param => $value ) {
+		foreach ( $params as $param ) {
```

Comment:
Onde mudou isto? Porque não é `foreach ( $params as $type => $value )`?

Reply:
Simples, pq quando eu tenho que por em vários postTypes ou taxonomias o campo o ACF, não funciona. 
Exemplo:
`$params =  [
			'post_type' => 'biography' ,
			'post_type' => 'exhibition',
			'post_type' => 'work',
		];`

Reply:
Mas onde é que foi mudada a estrutura do array `$params`? É que neste PR só mudas a forma como acedes aos conteúdos.

Reply:
English ppl. This is a public repository. @lzcalderaro you need a descriptive title and description.

Reply:
Use Google Translate 😛

Reply:
@lzcalderaro See my comment.

Reply:
hahaha Ok